src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
+    {ok, Blacklist} = couch_util:parse_term(BlacklistConfig),
+    Key = binary_to_list(KeyBin),
+    lists:member(Key, Blacklist).
```

Comment:
Like it. Thought, may be "disable.foo = true" or something else in imperative mood.